service/src/main/java/bio/terra/axonserver/service/cromwellworkflow/CromwellWorkflowService.java

```diff
@@ -280,6 +290,34 @@ public CromwellApiLabelsResponse getLabels(UUID workflowId)
         .labels(CROMWELL_CLIENT_API_VERSION, workflowId.toString());
   }
 
+  public Map<String, String> parseInputs(UUID workspaceId, String gcsPath, BearerToken token)
+      throws IOException, InvalidWdlException {
+    // 1) Get the WDL file and write it to disk
+    try (AutoDeletingTempFile tempWdlFile = new AutoDeletingTempFile("workflow-main-wdl", "wdl")) {
```

Comment:
nit: this will end up looking like workflow-main-wdloiasdh89yewf9hawefwdl, perhaps go with new AutoDeletingTempFile("workflow-main-", "-terra")?